addon/props/model_props.py

```diff
@@ -86,6 +86,21 @@ class SOURCEOPS_ModelProps(bpy.types.PropertyGroup):
         default=False,
     )
 
+    custom_transform_source: bpy.props.EnumProperty(
+        name='Transform Source',
+        description='Method of specifying $origin and $scale\nEither manually specified in this Panel, or via an object',
```

Comment:
You should end the first sentence with a period. The last sentence doesn't need one, as Blender adds its own.
I probably wouldn't capitalize the P in Panel.